component/loki/source/journal/types.go

```diff
@@ -0,0 +1,37 @@
+package journal
+
+import (
+	"time"
+
+	"github.com/grafana/agent/component/common/loki"
+	flow_relabel "github.com/grafana/agent/component/common/relabel"
+)
+
+// Arguments are the arguments for the component.
+type Arguments struct {
+	FormatAsJson   bool                   `river:"format_as_json,attr,optional"`
+	MaxAge         time.Duration          `river:"max_age,attr,optional"`
+	Path           string                 `river:"path,attr,optional"`
+	RelabelConfigs []*flow_relabel.Config `river:"rule,block,optional"`
```

Comment:
Do you think we should reuse the RelabelRules export here instead? Or still find it unwieldy?

Reply:
I can change, I think its functionally the same.